test/powershell/Language/Classes/Scripting.Classes.RunPath.Tests.ps1

```diff
@@ -0,0 +1,42 @@
+Describe "Script with a class definition run path" -Tags "CI" {
+
+    It "Script with a class definition can run from a path without a comma" {
+
+        $FilePath = '.\MyTest.ps1'
+
+        try {
+            'class MyClass { [string]$MyProperty }; $True' | Out-File -FilePath $FilePath
```

Comment:
Use `testdrive:` or `$testdrive` - then Pester will ensure the file is deleted when the test completes.